adding rss generator to generate the opml file (#28)
GoogleChromeLabs/privacy-sandbox-explainer-updates: README.md

    `python3 pseupdates.py --since=2022-04-01`

# RSS

> For RSS subscriber, you can generate a RSS OPML file and run the following:

Reviewer: ```suggestion
For users of RSS, you can generate a RSS OPML file by running the following:
```
